### DISCUSSION TITLE
Website changes suggestions

### DISCUSSION THREAD

**Discussion by Paul-Eraman-CoopCycle** [coopcycle/coopcycle-web#4584] [General]
Please write suggestions below!

**Comment by JeromeCROW** [comment]
https://coopcycle.org/cooperatives/

I would use another map type, or map surface. the current one looks weird, the buttons are unprecise, they don't name the coops involved and most importantly, the map is simply wrong. it lists marseille as member city, but there is no eraman for example. 

some of the photos on this page are ok, but they are not enough and some are rather terrible (mensakas, tricargo)
